Readme.md

```diff
@@ -1,13 +1,14 @@
 
 # escape-html
 
-  Escape HTML entities
+  Escape HTML special characters (syntax), replacing them with character references.
 
 ## Example
 
 ```js
 var escape = require('escape-html');
-escape(str);
+escape('<meta content="\'a\' & b" />');
+// &lt;meta content=&quot;&#39;a&#39; &amp; b&quot; /&gt;
```

Comment:
I feel like this is too noisy and a really weird thing for someone to actually want to escape? Especially with the trailing `/` which is XHTML syntax.